### PR TITLE
Azure right sizing

### PR DESCRIPTION
**Pull request by lbliii**
Here's a first pass at the Azure VM doc (based on the recommendations report one). Let me know if you'd like anything added/changed and I'll aim to get it updated by Friday.

### CODE REVIEW

**Comment by rclagett** [comment]
I agree with @CryptoPishka 's feedback. Number 1 could go in the opening paragraph somewhere (here's the full link - https://us.cloudwisdom.virtana.com/#/integrations/azure/new). Number 2 is blocked until we can figure out how to show that view (see #cw-team-content). Number 3 is a few minutes.

**Comment by rclagett** [comment]
I'll now go through this and provide feedback of my own if there is any!

File: content/right-sizing/reports-azure-right-sizing.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+### 1. Scope of Analysis
+
+The following scope criteria can be used to both include or exclude resources:
+
+- **Names**: Select resources by inserting a string.
+- **Virtual Machines**: Select resources from a dropdown.
+- **Attributes**: Select resources based on attributes; supports `any` and `all` statements.
+- **Tags**: Select resource based on tags; supports `any` and `all` statements.
+
+### 2. CPU and Memory Constraints
```
Comment (rclagett) [inline content/right-sizing/reports-azure-right-sizing.md:51]: Two ideas for this suggestion:
1. Mention the ability to set a target utilization for both CPU and memory utilization.
2. Mention how memory utilization can be set manually (Default memory utilization when diagnostic monitoring not enabled) if guest OS diagnostic metrics haven't been enabled and link to https://docs.metricly.com/integrations/microsoft-azure/azure-enable-guest-os-diagnostic/.

**Comment by rclagett** [comment]
Update: Number 2 isn’t necessary at this time as the view has not been introduced yet. The other feedback is still relevant.

**Comment by CryptoPishka** [comment]
looks good to me, ty !

**Pull request merged**